common.sh

```diff
@@ -50,10 +50,11 @@ NODES_FILE=${NODES_FILE:-"${WORKING_DIR}/ironic_nodes.json"}
 NODES_PLATFORM=${NODES_PLATFORM:-"libvirt"}
 MASTER_NODES_FILE=${MASTER_NODES_FILE:-"ocp/master_nodes.json"}
 
-export RHCOS_IMAGE_URL=${RHCOS_IMAGE_URL:-"https://releases-rhcos.svc.ci.openshift.org/storage/releases/ootpa/"}
+export RHCOS_INSTALLER_IMAGE_URL=$(jq -r '.baseURI' $GOPATH/src/github.com/openshift-metalkube/kni-installer/data/data/rhcos.json)
+export RHCOS_IMAGE_URL=${RHCOS_IMAGE_URL:-${RHCOS_INSTALLER_IMAGE_URL}}
 export RHCOS_INSTALLER_PIN=$(jq -r '.buildid' $GOPATH/src/github.com/openshift-metalkube/kni-installer/data/data/rhcos.json)
 export RHCOS_IMAGE_VERSION="${RHCOS_IMAGE_VERSION:-${RHCOS_INSTALLER_PIN}}"
-export RHCOS_IMAGE_FILENAME_OPENSTACK_GZ="$(curl ${RHCOS_IMAGE_URL}/${RHCOS_IMAGE_VERSION}/meta.json | jq -r '.images.openstack.path')"
+export RHCOS_IMAGE_FILENAME_OPENSTACK_GZ="$(curl ${RHCOS_IMAGE_URL}/meta.json | jq -r '.images.openstack.path')"
```

Comment:
Since RHCOS_IMAGE_URL now must contain the build version, I don't think we need RHCOS_IMAGE_VERSION at all anymore

get_images.sh should just use RHCOS_IMAGE_URL too